patches/chrome-browser-ui-content_settings-content_setting_image_model.cc.patch

```diff
@@ -15,7 +15,7 @@ index 4f8182d209673801407db8955c370a026511deed..55361a1022b7cb2b683c336894834b20
    for (auto type : kContentSettingImageOrder)
      result.push_back(CreateForContentType(type));
 +#if defined(BRAVE_CHROMIUM_BUILD)
-+  BraveGenerateContentSettingImageModels(result);
```

Comment:
let's move this to `chromium_src/chrome/browser/ui/content_settings/content_setting_image_model.cc` along with `+#include "brave/browser/ui/content_settings/brave_content_setting_image_models.h"`
And define these as `BRAVE_GENERATE_CONTENT_SETTING_IMAGE_MODELS` macro

Reply:
I'm wondering do we really need `BRAVE_CHROMIUM_BUILD` anywhere (cc @bridiver @simonhong) ?
we also probably can avoid patching by wrapping `GenerateContentSettingImageModels` with our override

Reply:
In this case (just adding our code), I think `BRAVE_CHROMIUM_BUILD` can be deleted.